pixart.js

```diff
@@ -1 +1,32 @@
+var setColorButton = document.querySelector("#set-color");
+var inputColor = document.querySelector("#color-field");
+var colorField = document.querySelector("#color-field");
+var brush = document.querySelector(".brush");
+var body = document.querySelector("body");
```

Comment:
This probably doesn't need to be stored in a variable. Also `document.body` is a quicker and more succinct way to target the body.